### DISCUSSION TITLE
Only one account per mail

### DISCUSSION THREAD

**Discussion by Tealk** [goauthentik/authentik#8933] [Q&A] [answered]
Hello everyone,

can someone tell me how to customize the registration process to check if there is already an account with the given email address?

**Comment by BeryJu** [comment, marked as answer]
Email addresses are already unique in the database, so when using a prompt field of the type `email` that uniqueness check is already done

**Reply by Tealk** [reply]
I recently deleted a whole bunch of spam accounts, because I thought that several users had used the same one. But I may have been wrong as they were very cryptic.

**Reply by BeryJu** [reply]
Actually I just realised I was wrong about this. We'll update the docs to show how this behaviour can be changed.
